extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -388,18 +401,9 @@ export class ModelingStore extends DisposableObject {
     });
   }
 
-  public setSelectedMethod(dbItem: DatabaseItem, methodSignature: string) {
+  public setSelectedMethod(dbItem: DatabaseItem, method: Method, usage: Usage) {
```

Comment:
Should we add a comment to this method to specify that the `method` and `usage` need to have been retrieved from the modeling store? Theoretically you could pass in a usage received from the webview which might cause other code relying on object referential identity to break.